### PR TITLE
Proofreading Jamie White's episode - 1:00 1:15

### PR DESCRIPTION
**Pull request by nicoder**

### CODE REVIEW

File: 5/jamie-white-captions.srt [hunk 1]
```diff
@@ -2969,7 +2969,8 @@ And here the inverse is host.
 
 700
 01:01:13,050 --> 01:01:15,960
-At this point, I'm looking at build fabrics
+At this point, I'm looking at
+building collaborators
```
Comment (nelstrom) [inline 5/jamie-white-captions.srt:2973]: Good catch! :+1:

File: 5/jamie-white-captions.srt [hunk 2]
```diff
@@ -3057,7 +3062,7 @@ so we call this the application adapter.
 
 721
 01:03:52,050 --> 01:03:54,290
-DN: Ah, forgot to save the universe.
+DN: Ah, forgot to save the inverse.
```
Comment (nelstrom) [inline 5/jamie-white-captions.srt:3065]: :+1:

**Comment by nelstrom** [comment]
Great job @nicoder, thanks!

**Comment by nelstrom** [comment]
I slightly changed the workflow this time and it looks like you missed the new instructions. I created a `proofreading-jamie-white-captions` branch, and added a few of my own corrections before inviting tech reviewers to give it the once over. That means I've got [one PR](https://github.com/fromPeerToPeer/captions/pull/27/files) where I can bring together all of the changes before merging back into master.

I didn't use the big green merge button for your PR, but did it on the command line instead.

**Comment by nicoder** [comment]
Oh yes, sorry, I should have read your email with more attention, it was not very long.

And the worst is I did see the name of the branch appear on the command line when I fetched from upstream so I should have looked into it.

hope the merge was not too complicated.

**Pull request closed without merging**